addons/sale_timesheet/models/account.py

```diff
@@ -96,7 +96,7 @@ def _timesheet_determine_sale_line(self, task, employee, project):
                     return map_entry.sale_line_id
                 if task.sale_line_id or project.sale_line_id:
                     return task.sale_line_id or project.sale_line_id
-        return self.env['sale.order.line']
+        return False
```

Comment:
I don't know which is better...

Reply:
Before we changed and improved features in Service Apps, this method was used in the write method and if this method `return self.env['sale.order.line']` then we could used methods with @api.model decorator since you have the model in variable. But here, this method is only used in compute methods then we don't need to return the model in this method.

Reply:
Ok